cli/tools/lint.rs

```diff
@@ -121,7 +121,8 @@ pub async fn lint(
       move |file_path| {
         let file_text = fs::read_to_string(&file_path)?;
 
-        // don't bother rechecking this file if it didn't have any diagnostics before
+        // don't bother rechecking this file if it didn't have any diagnostics
+        // before
```

Comment:
Not really a fan of the wrapping in these cases. I kind of wish there was a heiristic in this case where it would only wrap if the next line would be 30 or 40 characters. Comments going over the line width is ok IMO.